docs/install/local-docker.md

```diff
@@ -158,3 +158,7 @@ The Jupyter environment is pre-configured to work with the local MLRun and Nucli
 You can switch to a remote or managed MLRun cluster by editing the `mlrun.env` file in the Jupyter files tree.
 
 The artifacts and DB are stored under **/home/jovyan/data** (`/data` in Jupyter tree). 
+
```

Comment:
Note should be moved to top section after the other note about ports. Update to something more concise, like:
If you are running more than one instance of MLRun, change the exposed port.